<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageswitchFrame</class>
 <widget class="QWidget" name="pageswitchFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QToolButton" name="menuButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="captureButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="analysisButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Analysis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="InputButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>AppleGothic</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Strategy and Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
